src/update/metadataMerger.ts

```diff
@@ -46,7 +42,7 @@ export class MetadataMerger {
       classification: isSwap ? updateMetadata.classification : this.mergeClassification(oldMetadata.classification, updateMetadata.classification),
     };
     newMetadata.productBoundingBox = createBBoxString(newMetadata.footprint as Footprint);
-    newMetadata.sensors = this.polygonPartsToSensors(newMetadata.layerPolygonParts as FeatureCollection);
+    newMetadata.sensors = updateMetadata.sensors;
     return newMetadata;
```

Comment:
fix to test each case (use existing tests)

Reply:
fixed